sql/core/src/main/scala/org/apache/spark/sql/streaming/progress.scala

```diff
@@ -106,7 +109,9 @@ class StreamingQueryProgress private[sql](
   val eventTime: ju.Map[String, String],
   val stateOperators: Array[StateOperatorProgress],
   val sources: Array[SourceProgress],
-  val sink: SinkProgress) extends Serializable {
+  val sink: SinkProgress,
+  @JsonDeserialize(contentAs = classOf[GenericRowWithSchema])
```

Comment:
what does it mean?

Reply:
It means that it can assume that the rows are instances of `GenericRowWithSchema`.